README.md

```diff
@@ -0,0 +1,27 @@
+# OpenTelemetry C/C++
+[![Gitter chat][gitter-image]][gitter-url]
+
+[gitter-image]: https://badges.gitter.im/open-telemetry/opentelemetry-cpp.svg
+[gitter-url]: https://gitter.im/open-telemetry/opentelemetry-cpp?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+
+The C/C++ [OpenTelemetry](https://opentelemetry.io/) client.
+
+## Installation
+
+TBD
+
+## Quick Start
+
+TBD
+
+## Contributing
+
+See [CONTRIBUTING.md](CONTRIBUTING.md)
+
+## Release Schedule
+
+OpenTelemetry C/C++ is under active development.
```

Comment:
Is this supposed to be a combined library for both C and C++ users? Are C users/libraries willing to depend on a C++ runtime for their libraries/products? If this is not the case (which I hope), please do not mix C into the naming, and use just "OpenTelemetry C++"

Reply:
This is a "to-be-discussed" topic for our 1st community meeting.

Reply:
I could imagine the potential-future C API belongs in its own repository.

Reply:
Yes, we've discussed this during the community meeting.
For now we will use this repo for the C++ SDK, and build a C API (no exception, no STL in the function signature) on top of the C++ SDK.
Later we might want to split the C API to its own repo.